packages/node-cli/src/commands/AddDependenciesCommandGenerator.js

```diff
@@ -69,18 +71,33 @@ module.exports = class AddDependenciesCommandGenerator extends BaseCommandGenera
 		return answers;
 	}
 
-	_executeAction(answers) {
-		const executionContext = new SDKExecutionContext({
-			command: this._commandMetadata.sdkCommand,
-			params: answers,
-			flags: [COMMAND_OPTIONS.ALL],
-			requiresContextParams: true
-		});
-
-		return executeWithSpinner({
-			action: this._sdkExecutor.execute(executionContext),
-			message: TranslationService.getMessage(MESSAGES.ADDING_DEPENDENCIES),
-		});
+	async _executeAction(answers) {
+		try {
+			const executionContext = new SDKExecutionContext({
+				command: this._commandMetadata.sdkCommand,
+				params: answers,
+				flags: [COMMAND_OPTIONS.ALL],
+				requiresContextParams: true
+			});
+
+			const operationResult = await executeWithSpinner({
+				action: this._sdkExecutor.execute(executionContext),
+				message: TranslationService.getMessage(MESSAGES.ADDING_DEPENDENCIES),
+			});
+
+			return operationResult.status === SDKOperationResultUtils.SUCCESS
+				? ActionResult.Builder
+					.success()
+					.withData(operationResult.data)
```

Comment:
Can withData function set successful status so we don't need to call success() additionally? Is there any case withData will need to be combined with error status?